src/session.ts

```diff
@@ -520,20 +527,87 @@ function dtmf(this: WebPhoneSession, dtmf: string, duration = 100, interToneGap
 
 /*--------------------------------------------------------------------------------------------------------------------*/
 
-function hold(this: WebPhoneSession): Promise<any> {
+async function sendReinvite(this: WebPhoneSession, options: any = {}): Promise<any> {
+    if (options === void 0) {
```

Comment:
Options will be defaulted already in function args definition

Reply:
Sure will change this.